app/values/health_report/derivative_check.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+class HealthReport::DerivativeCheck
+  def self.for(resource)
+    new(resource: resource)
+  end
+
+  attr_reader :resource
+  def initialize(resource:)
+    @resource = resource
+  end
+
+  def valid?
+    true
+  end
+
+  def type
+    I18n.t("health_status.derivative_check.type")
+  end
+
+  def status
+    @status ||=
+      if resource.is_a?(FileSet)
+        file_set_status
+      elsif children_errored?
+        :needs_attention
+      elsif children_processing?
+        :in_progress
+      else
+        :healthy
+      end
+  end
+
+  def file_set_status
+    if resource.error_message.present?
+      :needs_attention
+    elsif resource.processing_status == "in process"
+      :in_progress
+    else
+      :healthy
+    end
+  end
+
+  def summary
+    if resource.respond_to?(:member_ids)
+      I18n.t("health_status.derivative_check.summary.#{status}")
+    else
+      I18n.t("health_status.derivative_check.summary.self.#{status}")
```

Comment:
They don't look different? What am I missing?

Reply:
To be more specific, it looks like the translations are identical in both cases. I think we don't need a code branch here, and we could just have one set of translations.

Reply:
Oh I copied it from the others. I can remove it.